packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -101,6 +101,7 @@ class _WidgetInspectorState extends State<WidgetInspector>
     final List<DiagnosticsNode> children = object.debugDescribeChildren();
     for (int i = children.length - 1; i >= 0; i -= 1) {
       final DiagnosticsNode diagnostics = children[i];
+      assert(diagnostics != null);
```

Comment:
of course, the other fix for this issue would be to just continue if children[i]  == null as clearly a null DiagnosticsNode child doesn't need to be hit tested against.